modules/fedimint-mint-client/src/lib.rs

```diff
@@ -92,6 +93,14 @@ pub struct OOBNotes(Vec<OOBNotesData>);
 enum OOBNotesData {
     Notes(TieredMulti<SpendableNote>),
     FederationIdPrefix(FederationIdPrefix),
+    /// Invite code to join the federation by which the e-cash was issued
+    ///
+    /// Introduce in 0.3.0
+    Invite {
```

Comment:
If this is present the `FederationIdPrefix` is redundant, though we might start to omit it in some future. Though we might already change code that uses `FederationIdPrefix` to use see if `Invite` is available.

Reply:
Is this a documentation request?

Reply:
No. Code request, unless I'm missing something.

Reply:
https://github.com/fedimint/fedimint/pull/4231#discussion_r1477646687